### PR TITLE
Fix plain body encoding

### PR DESCRIPTION
**Pull request by soffes**
Before this bodies would appear in JSON as `"Optional(\"Some text\")"`. Now they appear as `"Some text"`.

### CODE REVIEW

File: DVR/URLRequest.swift [hunk 1]
```diff
@@ -41,6 +41,8 @@ extension NSMutableURLRequest {
             allHTTPHeaderFields = headers
         }
 
-        HTTPBody = Interaction.dencodeBody(dictionary["body"], headers: allHTTPHeaderFields)
+		if let body = dictionary["body"] {
```
Comment (ayanonagon) [inline DVR/URLRequest.swift:44]: Fix your tab spacing plz :pray:

File: DVR/Tests/SessionTests.swift [hunk 2]
```diff
@@ -31,18 +31,42 @@ class SessionTests: XCTestCase {
         waitForExpectationsWithTimeout(1, handler: nil)
     }
 
+	func testTextPlayback() {
```
Comment (ayanonagon) [inline DVR/Tests/SessionTests.swift:34]: Same

File: DVR/Cassette.swift [hunk 3]
```diff
@@ -55,14 +55,14 @@ extension Cassette {
 
 private extension NSURLRequest {
     func hasHTTPBodyEqualToThatOfRequest(request: NSURLRequest) -> Bool {
-        if let body1 = self.HTTPBody,
+        guard let body1 = self.HTTPBody,
             body2 = request.HTTPBody,
             encoded1 = Interaction.encodeBody(body1, headers: self.allHTTPHeaderFields),
-            encoded2 = Interaction.encodeBody(body2, headers: request.allHTTPHeaderFields) {
+            encoded2 = Interaction.encodeBody(body2, headers: request.allHTTPHeaderFields)
+		else {
```
Comment (ayanonagon) [inline DVR/Cassette.swift:62]: Spaces please!

File: DVR/Cassette.swift [hunk 4]
```diff
@@ -55,14 +55,14 @@ extension Cassette {
...
+			return self.HTTPBody == request.HTTPBody
+		}
 
-                return encoded1.isEqual(encoded2)
-        } else {
-            return self.HTTPBody == request.HTTPBody
-        }
+		return encoded1.isEqual(encoded2)
```
Comment (ayanonagon) [inline DVR/Cassette.swift:66]: Ditto.

**Comment by ayanonagon** [comment]
Tabs to spaces, then :green_apple:. Thanks @soffes!

**Comment by soffes** [comment]
Closing since this is part of #31

**Pull request closed without merging**